services/director-v2/src/simcore_service_director_v2/modules/comp_scheduler/_scheduler_factory.py

```diff
@@ -24,39 +20,19 @@ async def create_from_db(app: FastAPI) -> BaseCompScheduler:
         msg = "Database connection is missing. Please check application configuration."
         raise ConfigurationError(msg)
     db_engine = app.state.engine
-    runs_repository = CompRunsRepository.instance(db_engine)
-
-    # get currently scheduled runs
-    runs: list[CompRunsAtDB] = await runs_repository.list(
-        filter_by_state=SCHEDULED_STATES
-    )
-
-    _logger.debug(
-        "Following scheduled comp_runs found still to be scheduled: %s",
-        runs if runs else "NONE",
-    )
 
     with log_context(
         _logger, logging.INFO, msg="Creating Dask-based computational scheduler"
     ):
         app_settings: AppSettings = app.state.settings
-        return DaskScheduler(
+        scheduler = DaskScheduler(
             settings=app_settings.DIRECTOR_V2_COMPUTATIONAL_BACKEND,
             dask_clients_pool=DaskClientsPool.instance(app),
             rabbitmq_client=get_rabbitmq_client(app),
             rabbitmq_rpc_client=get_rabbitmq_rpc_client(app),
             redis_client=get_redis_client_manager(app).client(RedisDatabase.LOCKS),
             db_engine=db_engine,
-            scheduled_pipelines={
-                (r.user_id, r.project_uuid, r.iteration): ScheduledPipelineParams(
-                    cluster_id=(
-                        r.cluster_id if r.cluster_id is not None else DEFAULT_CLUSTER_ID
-                    ),
-                    run_metadata=r.metadata,
-                    mark_for_cancellation=r.cancelled,
-                    use_on_demand_clusters=r.use_on_demand_clusters,
-                )
-                for r in runs
-            },
             service_runtime_heartbeat_interval=app_settings.SERVICE_TRACKING_HEARTBEAT,
         )
+        await scheduler.restore_scheduling_from_db()
```

Comment:
OK, so here is where you restore the `previous state`, right? So the idea is that after init, I recover where I left it last time and continue ...

Reply:
yes.